scripts/publish-release.sh

```diff
@@ -4,20 +4,20 @@
 set -ex
 
 ## Obtain/check npm access
-npm whoami
-if [ "$?" != "0" ];
+if ! npm whoami;
   then
     npm login
 fi
 node ./scripts/npm-access.js
 
 ## Publish packages to npm
 lerna version
```

Comment:
The release branch is the active branch at this point, and I'm unsure about the behavior of `lerna version` at this point.
- previously the command ran on active branch `develop`, which had an `origin/develop` remote. Will it work without the `origin/release/EPOCH` remote?
- assuming `version` works, the changes committed to the remote branch gets merged into `origin/master`, however `origin/develop` still needs reconciliation. Note, local `develop` and `origin/develop` could have diverged if someone merged a PR during the release process.

Reply:
[lerna readme](https://github.com/lerna/lerna/tree/main/commands/version#readme)

It says that it bumps the packages "that have been updated since the previous tagged release". It doesn't seem like it cares about what branch you are on or take that into account at all.

And yep, `develop` could have changed. So it is possible someone will have to resolve a conflict. Is there any alternative to this?